data/js/configuring-data.md

```diff
@@ -0,0 +1,164 @@
+# Configuring Data
+
+###### The `api.json` and `api-*.json` files allow us to configure the accessible paths for each data service.
+
+<!-- <article id="undestanding-configuration-files"> -->
+
+## Understanding configuration files
+
+By default WeDeploy Data service is going to use all the JSON files starting with `api-*` and also the file `api.json`.
+
+These files are used to help you manage features such as path validation, authentication, and params validation.
+
+The api JSON files are located at the same path of the `./container.json` and are used following the ordering filesystem.
+
+By the default, if you don't add any api.json file all the collections are going to be exposed.
```

Comment:
Yes, if we create this container with github on Deployment. But i believe in this case we don't have the `By the default` behavior, as it comes from boilerplate-data that is overridden in built containers with user's files (or lack thereof).